pkg/ddevapp/config.go

```diff
@@ -102,12 +102,14 @@ func NewConfig(AppRoot string, provider string) (*Config, error) {
 	}
 	// Load from file if available. This will return an error if the file doesn't exist,
 	// and it is up to the caller to determine if that's an issue.
-	err := c.Read()
-	if err != nil {
-		return c, fmt.Errorf("Unable to read config.yaml, %v, err=%v", c.ConfigPath, err)
+	if _, err := os.Stat(c.ConfigPath); !os.IsNotExist(err) {
+		err = c.Read()
+		if err != nil {
+			return c, fmt.Errorf("%v[configPath] exists but cannot be read: %v[err]", c.ConfigPath, err)
```

Comment:
one of my poorly-formatted suggestions missed the roll-back. sorry for the confusion here.

Reply:
Thanks for the sharp eye, fixed.